src/index.ts

```diff
@@ -79,11 +79,11 @@ class LiskMigrator extends Command {
 			description:
 				'File path to write the genesis block json. If not provided, it will default to cwd/genesis_block.json.',
 		}),
-		'lisk-core-path': flagsParser.string({
+		'lisk-core-v3-data-path': flagsParser.string({
 			char: 'd',
 			required: false,
 			description:
-				'Path where the lisk-core instance is running. Current directory will be considered the default if not provided.',
+				'Path where the lisk-core v3.x instance is running. Current home directory will be considered the default if not provided.',
```

Comment:
```suggestion
				'Path where the lisk-core v3.x instance is running. The current home directory will be considered the default directory if not otherwise provided.',
```